cmd/config.go

```diff
@@ -172,6 +174,10 @@ func initConfig(objAPI ObjectLayer) error {
 				return err
 			}
 		}
+
+		// Watch config for changes and reloads them i
+		go watchConfig(objAPI, configFile, loadConfig)
```

Comment:
Shouldn't we also watch for config file in case of distributed XL ?

Reply:
watchConfig is only etcd call, it should be renamed.